### PR TITLE
Update after PXC 1.8.0 release and prepare for 1.9.0

### PR DESCRIPTION
**Pull request by tplavcic**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| scheduled-backup | passed |
| storage | failed |
| haproxy | failed |
| big-data | passed |
| upgrade-haproxy | failed |
| recreate | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| security-context | failed |
| restore-to-encrypted-cluster | passed |
| init-deploy | failed |
| self-healing-advanced | passed |
| demand-backup | passed |
| upgrade-proxysql | failed |
| limits | failed |
| monitoring-2-0 | failed |
| operator-self-healing | passed |
| affinity | failed |
| pitr | passed |
| one-pod | passed |
| smart-update | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | failed |
| demand-backup-encrypted-with-tls | passed |
| users | passed |
| upgrade-consistency | failed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/862/commits/63d6c2a405a617a6e5e6332a0db54b5cbcd8b5c1
image: `perconalab/percona-xtradb-cluster-operator:PR-862-63d6c2a4`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scaling | passed |
| upgrade-haproxy | failed |
| scheduled-backup | passed |
| haproxy | failed |
| recreate | passed |
| storage | failed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| security-context | failed |
| self-healing | passed |
| init-deploy | failed |
| self-healing-advanced | passed |
| demand-backup | passed |
| upgrade-proxysql | failed |
| limits | failed |
| operator-self-healing | failed |
| monitoring-2-0 | failed |
| pitr | passed |
| smart-update | passed |
| affinity | failed |
| one-pod | passed |
| auto-tuning | passed |
| demand-backup-encrypted-with-tls | passed |
| proxysql-sidecar-res-limits | failed |
| upgrade-consistency | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/862/commits/23ae0b0b59cfa04147f5a9f81c6b059ed754bd5c
image: `perconalab/percona-xtradb-cluster-operator:PR-862-23ae0b0b`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| storage | failed |
| scheduled-backup | passed |
| scaling | passed |
| big-data | passed |
| upgrade-haproxy | passed |
| haproxy | passed |
| init-deploy | failed |
| scaling-proxysql | passed |
| security-context | failed |
| upgrade-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| self-healing | passed |
| smart-update | passed |
| demand-backup | passed |
| self-healing-advanced | passed |
| limits | failed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| affinity | failed |
| one-pod | passed |
| pitr | passed |
| auto-tuning | passed |
| upgrade-consistency | passed |
| proxysql-sidecar-res-limits | failed |
| demand-backup-encrypted-with-tls | failed |
| users | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/862/commits/c23c0562b969df96ac036fdf8606eff72a7181aa
image: `perconalab/percona-xtradb-cluster-operator:PR-862-c23c0562`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| scheduled-backup | passed |
| storage | failed |
| recreate | passed |
| upgrade-haproxy | passed |
| big-data | passed |
| haproxy | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| security-context | failed |
| restore-to-encrypted-cluster | passed |
| upgrade-proxysql | passed |
| limits | failed |
| self-healing | passed |
| demand-backup | passed |
| smart-update | passed |
| monitoring-2-0 | passed |
| self-healing-advanced | passed |
| affinity | failed |
| one-pod | passed |
| operator-self-healing | passed |
| auto-tuning | passed |
| pitr | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| upgrade-consistency | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-self | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/862/commits/28d88750c5f9a3223eb3098b7520e2eca2c72f2f
image: `perconalab/percona-xtradb-cluster-operator:PR-862-28d88750`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade-haproxy | passed |
| scaling | passed |
| big-data | passed |
| storage | passed |
| haproxy | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| self-healing | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| limits | failed |
| self-healing-advanced | passed |
| smart-update | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| affinity | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| upgrade-consistency | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/862/commits/40f2a632ddd2300dafcfe1b46a90c149767b4223
image: `perconalab/percona-xtradb-cluster-operator:PR-862-40f2a632`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scheduled-backup | passed |
| haproxy | passed |
| storage | passed |
| scaling | passed |
| recreate | passed |
| upgrade-haproxy | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| upgrade-proxysql | passed |
| security-context | passed |
| restore-to-encrypted-cluster | passed |
| limits | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| smart-update | passed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| pitr | passed |
| users | passed |
| upgrade-consistency | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/862/commits/68f7eca7749716b475e55e714defb1ec62e7b35a
image: `perconalab/percona-xtradb-cluster-operator:PR-862-68f7eca7`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| pitr | passed |
| demand-backup-encrypted-with-tls | passed |
| upgrade-haproxy | passed |
| scheduled-backup | passed |
| big-data | passed |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| scaling | passed |
| scaling-proxysql | passed |
| security-context | passed |
| haproxy | passed |
| init-deploy | passed |
| limits | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| upgrade-proxysql | passed |
| tls-issue-cert-manager-ref | passed |
| smart-update | passed |
| upgrade-consistency | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/862/commits/68f7eca7749716b475e55e714defb1ec62e7b35a
image: `perconalab/percona-xtradb-cluster-operator:PR-862-68f7eca7`

File: e2e-tests/upgrade-consistency/run [hunk 1]
```diff
@@ -141,17 +141,25 @@ main() {
...
     kubectl_bin patch pxc "$cluster" --type=merge --patch '{
         "spec": {"crVersion":"1.8.0"}
     }'
     wait_cluster_consistency "$cluster" 3 2
     wait_for_sts_generation "$cluster-pxc" "8" "1"
 
-    compare_kubectl service/$cluster-pxc "-180"
-    compare_kubectl service/$cluster-proxysql "-180"
-    compare_kubectl statefulset/$cluster-pxc "-180"
-    compare_kubectl statefulset/$cluster-proxysql "-180"
```
Comment (delgod) [inline e2e-tests/upgrade-consistency/run:154]: why these lines was removed?

**Review by delgod — CHANGES_REQUESTED** [review]

File: e2e-tests/upgrade-consistency/run (hunk 1 above)
Comment (tplavcic) [reply e2e-tests/upgrade-consistency/run]: By mistake! Made another commit to return them.

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| big-data | passed |
| upgrade-haproxy | passed |
| scaling | passed |
| haproxy | passed |
| scheduled-backup | passed |
| storage | passed |
| init-deploy | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| upgrade-proxysql | passed |
| limits | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scaling-proxysql | passed |
| operator-self-healing | passed |
| one-pod | passed |
| security-context | passed |
| auto-tuning | passed |
| pitr | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| demand-backup-encrypted-with-tls | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/862/commits/91d5f29f659c2a193d7da95513f0a30b68c5f9f2
image: `perconalab/percona-xtradb-cluster-operator:PR-862-91d5f29f`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |
| big-data | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| pitr | passed |
| demand-backup-encrypted-with-tls | passed |
| scaling | passed |
| scaling-proxysql | passed |
| security-context | passed |
| storage | passed |
| self-healing | passed |
| upgrade-haproxy | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| haproxy | passed |
| init-deploy | passed |
| limits | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |
| upgrade-consistency | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/862/commits/91d5f29f659c2a193d7da95513f0a30b68c5f9f2
image: `perconalab/percona-xtradb-cluster-operator:PR-862-91d5f29f`

**Pull request merged**